tests/pytests/integration/daemons/test_masterapi.py

```diff
@@ -120,3 +121,23 @@ def test_autosign_grains_fail(
     )  # get minion to try to authenticate itself again
     assert salt_minion.id not in salt_key_cli.run("-l", "acc")
     assert salt_minion.id in salt_key_cli.run("-l", "un")
+
+
+@pytest.mark.skip_unless_on_linux
+@pytest.mark.slow_test
+def test_ufw_allow(salt_master, grains):
+    if grains["os_family"] != "Debian":
+        pytest.skip("Only runs on Debian family.")
+
+    expected_output = """Skipping adding existing rule
+Skipping adding existing rule (v6)
+    """
+
+    proc = subprocess.Popen(
+        "ufw allow salt\n", shell=True, stdin=subprocess.PIPE, stdout=subprocess.PIPE
```

Comment:
Question: If this is a package change, wouldn't the test need to be in the package test suite?

Reply:
You raise an interesting point.  I wanted to be able to test this without things having been packaged, but I should probably add a test there too, or move the test there.

Will look into it.